client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -27,11 +27,109 @@
         style="max-height: 52vh;"
       />
     </b-form-group>
+
+    <hr>
+
+    <div>
+      <b-form-group
+        label-size="lg"
+      >
+        <template #label>
+          <div
+            class="d-flex"
+          >
+            {{ $t('record.fieldConditions.label') }}
+
+            <b-button
+              variant="link"
+              class="p-0 ml-1"
+              :disabled="block.options.fields.filter(f => !f.isRequired).length === block.options.fieldConditions.length"
+              @click="addRules"
+            >
+              {{ $t('record.fieldConditions.action') }}
+            </b-button>
+
+            <b-button
+              variant="link"
+              :href="`${documentationURL}#value-sanitizers`"
+              target="_blank"
+              class="p-0 ml-auto"
+            >
+              {{ $t('record.fieldConditions.help') }}
+            </b-button>
+          </div>
+        </template>
+      </b-form-group>
+
+      <b-table-simple
+        v-if="block.options.fieldConditions.length"
+        borderless
+      >
+        <b-thead>
+          <b-tr>
+            <b-th
+              class="text-primary"
+            >
+              {{ $t('record.fieldConditions.field') }}
+            </b-th>
+            <b-th
+              class="text-primary"
+            >
+              {{ $t('record.fieldConditions.condition') }}
+            </b-th>
+            <b-th />
+          </b-tr>
+        </b-thead>
+
+        <b-tbody>
+          <b-tr
+            v-for="(condition, i) in block.options.fieldConditions"
+            :key="i"
+          >
+            <b-td>
```

Comment:
i suggest you add a "width: 33%; min-width: 200px" here

Also look at how namespace sidebar handles long names inside vue-select (elipsis)